application/data-application.R

```diff
@@ -316,10 +412,19 @@ set.seed(123456789)
 (noeff_aemet_3 <- flm_test(X = aemet_temp_pred, Y = aemet_temp_resp,
                            B = B, est_method = "fpcr_l2", beta0 = 0,
                            save_fit_flm = FALSE, save_boot_stats = FALSE))
+
+# Kokoszka et al.
 set.seed(123456789)
 (kok_aemet <- kokoszka_test(X_fpc = flmfr_aemet_1$fit_flm$X_fpc,
                             Y_fpc = flmfr_aemet_1$fit_flm$Y_fpc))
 
+# Lee et al.: statistic = 9112344, p-value < 2.2e-16
+set.seed(123456789)
+(lee_aemet <-
+    FMDD_test(X_fdata = aemet_temp_pred, Y_fdata = aemet_temp_resp, B = B))
+
+
+
 # Smooth data
```

Comment:
For the smooth test, kindly change `fda.usc::min.np` by `fda.usc::optim.np` (check https://github.com/moviedo5/fda.usc/blob/master/NEWS).